unfurl/configurators/octodns.py

```diff
@@ -0,0 +1,194 @@
+import logging
+import os
+from collections import MutableMapping
+from contextlib import contextmanager
+from dataclasses import dataclass
+from pathlib import Path
+
+from octodns.manager import Manager
+from ruamel.yaml import YAML
+
+from unfurl.configurator import Configurator
+from unfurl.eval import map_value
+from unfurl.job import ConfigTask
+from unfurl.projectpaths import WorkFolder
+
+log = logging.getLogger(__file__)
```

Comment:
use `task.logger` instead of log (you do with some of the code) and if there are place you can't use `unfurl` instead `__file__` (which expands to an absolute path)

Reply:
Ok, `task.logger` is used everywhere. Btw, I made a mistake before. Instead of `__file__` I meant to use `__name__`. That would give us string `unfurl.configurators.octodns` which would be better than `unfurl` because we (or some 3r party) could specify logs behaviour more precisely.